app/routes/git_tokens.py

```
@@ -4,340 +4,352 @@
-
-    token_len = len(token)
-
-    if token_len <= 8:
-        return "*" * token_len
-
-    prefix = token[:4]
-    suffix = token[-4:]
-    middle_mask = "*" * (token_len - 8)
-
-    return f"{prefix}{middle_mask}{suffix}"
+	"""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)